.github/workflows/docker-tag-latest-release.yml

```diff
@@ -0,0 +1,62 @@
+name: Docker Tag latest-release
+
+# Run every couple of days
+on:
+  schedule:
+  - cron: "* */48 * * *"
+
+jobs:
+  # This pulls all containers for the last known release tag, and tags as latest-release or x-latest-release for shared repo
+  tagLatestRelease:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Pull the code so we have git metadata
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # This action will "unshallow" so we have all tag info
+    - name: Get latest tag
+      id: tagger
+      uses: jimschubert/query-tag-action@v1
+      with:
+        include: 'v*'
+        exclude: '*-rc*'
+        commit-ish: 'HEAD~'
+
+    - name: DockerHub Login
+      run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
```

Comment:
note to reviewer, I'd recommend trying a similar command here in a private repo to become familiar with it. This follows GitHub Workflow documentation by passing creds into supported stdin.